index.js

```diff
@@ -972,6 +972,22 @@
     return either instanceof Left ? l(either.value) : r(either.value);
   });
 
+  //# maybeToEither :: a -> Maybe b -> Either a b
+  //.
+  //. Converts a Maybe value to an Either value, using the supplied
+  //. parameter as the Left value if the Maybe is Nothing.
+  //.
+  //. ```javascript
+  //. > S.maybeToEither('Expecting an integer', S.parseInt(10, 'xyz'))
+  //. Left("Expecting an integer")
+  //.
+  //. > S.maybeToEither('Expecting an integer', S.parseInt(10, '42'))
+  //. Right(42)
+  //. ```
+  S.maybeToEither = def('maybeToEither', [a, Maybe], function(x, maybe) {
+    return maybe instanceof Nothing ? Left(x) : maybe.chain(Right);
```

Comment:
I was wondering about that. I think I originally had `Right(maybe.chain(R.identity))`. Didn't realize I was allowed to reach into the `maybe` :)

> f must return a value of the same Chain

Interesting, since it doesn't have to return a `chain`able value at all, right (as when chaining with identity)?